addons/l10n_in_pos/models/account_move.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _compute_l10n_in_state_id(self):
+        res = super()._compute_l10n_in_state_id()
+        to_compute = self.filtered(lambda m: m.country_code == 'IN' and not m.l10n_in_state_id and m.journal_id.type == 'general')
+        pos_session_moves = self.env['pos.session'].sudo().search_read(
```

Comment:
We can not move the l10n_in_pos_session_ids field to l10n_in_pos?  The ORM might handle it well.

Reply:
I don't think so because the new field needs an update of the module.
tell me if I am wrong.

Reply:
@jva-odoo It does not change the database model.  It is like a non-stored field.

Reply:
@jco-odoo, I have moved the l10n_in_pos_session_ids field from l10n_in_reports_gstr_pos to l10n_in_pos. I have updated the code accordingly.